### PR TITLE
Make docs build again cleanly

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 714866c2e23f6e4d98004d5df52f32b6eb5bdebf:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d6ada8e7911e910fa2ce67f12dcced7d49d27e0b:

Everything seems all right :smile: .
*CFME QE Bot*

File: utils/__init__.py [hunk 1]
```diff
@@ -185,7 +189,10 @@ def deferred_verpick(version_d):
 
     @classproperty
     def getter(self):
-        return _version_pick(version_d)
+        if on_rtd:
```
Comment (RonnyPfannschmidt) [inline utils/__init__.py]: that one is going to be trouble

Comment (psav) [inline utils/__init__.py]: In what way?

Comment (RonnyPfannschmidt) [inline utils/__init__.py]: version picks return varying types, returning a string in the rtd case has breakage potential

**Comment by dajoRH** [comment]
Lint report for commit 05c657e12478fd52ac08b5cbf90e33e8aa0822a5:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 45ae67b603e5bdf799610a01dc9e49dc34112c84:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5cfb03750dffe21be018bba6344a8b1f70b7f90e:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/automate/explorer.py [hunk 2]
```diff
@@ -296,12 +296,15 @@ def is_enabled(self):
...
     @classproperty
     def default(cls):
-        if not hasattr(cls, "_default_domain"):
-            cls._default_domain = version.pick({
-                version.LOWEST: None,
-                '5.3': cls('Default')
-            })
-        return cls._default_domain
+        if on_rtd:
+            return "Collapsed at runtime"
```
Comment (mfalesni) [inline cfme/automate/explorer.py]: maybe this one should return `cls('Default')` as does the line 305? Otherwise I am fine ith merging this.

**Comment by dajoRH** [comment]
Lint report for commit 344034e9e5bc176064a199bfcbba505f05a4e9a8:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**